test/helper.js

```diff
@@ -58,6 +62,36 @@ class FakeRoles {
   }
 }
 
+class FakePublisher {
+  constructor() {
+    this.clients = [];
+  }
+
+  async clientCreated(clientId) {
```

Comment:
Note that all of these functions take an object containing the arguments, so `async clientCreated({clientId})`

Reply:
Noted!